Add playback commands

ch33kaboo opened the pull request:
This is the implementation of the feature request #35 (I opened few hours earlier).
Adding commands to the plugin is useful because the user can control their playback without touching the mouse, they can use the command palette or assign hotkeys (keyboard shortcuts) to the commands.
Here is a breakdown of the changes:
- I defined `this.addCommand({})` for three different commands (play next track, play previous track, and toggle play/pause)
- to play next track and previous track, I just used `this.next()` and `this.previous()` respectively
- to play/pause current track, I added a variable to store the track state (whether it is playing or paused) and then make the command toggle the state

I tested the changes and they all work fine
![image](https://github.com/andrewmcgivery/obsidian-soundscapes/assets/100171494/c20f778b-9967-411f-afe7-27b87027b3c8)

ch33kaboo committed:
add commands to play previous & next track

ch33kaboo committed:
add commands to play/pause current track

andrewmcgivery commented on main.ts:
```diff
@@ -103,6 +106,37 @@ export default class SoundscapesPlugin extends Plugin {
+			id: "Play/Pause",
+			name: "Play/Pause current track",
+			callback: () => {
+				if (isTrackPlaying) {
```
Instead of introducing a new variable, you should be able to use:

```
if(this.localPlayerStateObservable.getValue().playerState === PLAYER_STATE.PLAYING)
```

ch33kaboo committed:
fix code for play/pause current track

in the previous commit, I was introducing a new global variable to know the state of the current track (whether it is playing or paused) but it seems like there is another way to do it without adding a new variable

ch33kaboo commented:
hello @andrewmcgivery , I didn't know there was a simple way to know if the track is playing or not and that's why I introduced a new variable.
Anyway, I fixed the code by using `if(this.localPlayerStateObservable.getValue().playerState === PLAYER_STATE.PLAYING)` as you mentioned, I tested it and it works all fine. 👍👍
